bundles/route/custom_route_generator.rst

```diff
@@ -0,0 +1,73 @@
+Custom Route-Generator
+======================
+
+The RouteBundle enables you to define custom Route-Generator.
+This Generator will be called everytime you create or update
+a route for any entity. For each entity you can define which
```

Comment:
`for the specified entity`? The point is that it is only called for one type of entities.